.env.example

```diff
@@ -1,5 +1,8 @@
 PORT=8081
-
+# Auth0
+SECRET=8caea1f31b3228a9e5a4b3438a956b2ce5cef2e8c9b788c63fdb959240c9887b
```

Comment:
This looks like real env vars, remember this file is just a example, but no real data must be placed here

Reply:
ok